learn/guides/ballerina-shell.md

```diff
@@ -161,26 +193,55 @@ The example below modifies the `sum()`function.
 =$ function sum(float a, float b) returns float => a + b 
                                                                                                                                                             
 =$ /dclns                                                                                                                                                                                                           
-| ('sum) function sum(float a, float b) returns float => a + b;
+| (sum) function sum(float a, float b) returns float => a + b;
+| (Person) type Person record {string name;int age;};
 ```
 
+Now, `sum()` is a function accepting float parameters and returning a float value.
 
-Now, `sum()` is an expression-bodied function accepting float parameters and returning a float value.
+>**Caution:** Modifying definitions can cause undefined behavior in some cases.
 
->**Caution:** Modifying definitions in incompatible ways will lead to undefined behavior.
+### Loading Definitions from a File
 
-## Loading Definitions from a File
-
-If you have any definitions in source files, you can load these definitions to the REPL through the `/file <FILE_PATH>` command. 
+If you have any definitions in source files, you can load these definitions to the Shell through the `/file <FILE_PATH>` command. 
 
 If the source file contains a main function, the Shell will disregard it.
 
+Consider following sample `text.bal` file
```

Comment:
```suggestion
For example, see the sample `text.bal` file below.
```

Reply:
Updated in 2a10777065a5455f4155b3fbdd75b7a70f169c75